### ISSUE TITLE
jrubyJar shouldn't require the jar task to execute

### ISSUE THREAD

**Issue by rtyler** [jruby-gradle/jruby-gradle-jar-plugin#44]
With @mkristian's recent changes being merged, it is now impossible to build a `jrubyJar` (fat jar) without also producing a jar artifact.

```
:jar SKIPPED
:jrubyJar

FAILURE: Build failed with an exception.

* What went wrong:
Cannot expand ZIP '/home/tyler/source/lookout/metron/build/libs/metron-3.0+0fd969d.jar' as it does not exist.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.
```

This dependency link should be broken and the `jrubyJar` task should be able to contruct its tree on its own without requiring that the `jar` task also execute.

In effect, I should be able to disable the `jar` task and still execute `jrubyJar`

**Comment by mkristian** [comment]
having the ```JRubyJar``` from #47 we also need to "extend" project to allow
```
jrubyJar {
  jruby {
    initScript( 'app.rb' )
  }
}
```

but on top we want to build "libraries" like
```
jar {
  jruby {
    defaultGems( 'app.rb' )
  }
}
```

the first on is similar to what we have with ```jrubyExec``` but the second needs to extend or configure  the ```Jar``` tasks to include those extra resources

**Comment by mkristian** [comment]
fixed with #47
